From 5837dc7a746e9b11dea6332a25df77007d59b02a Mon Sep 17 00:00:00 2001
From: Gabriele Modena <gabriele.modena@gmail.com>
Date: Wed, 23 Dec 2020 00:08:41 +0100
Subject: [PATCH] Attempt to address a style issue in nvec_power.c

When running checkpatch.pl on nvec_power.c I noticed
that it contained a style issue. This patch
attempts to address it by moving const after static.

Signed-off-by: Gabriele Modena <gabriele.modena@gmail.com>
---
 drivers/staging/nvec/nvec_power.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/nvec/nvec_power.c b/drivers/staging/nvec/nvec_power.c
index 0e861c4bfcbf..b1ef196e1cfe 100644
--- a/drivers/staging/nvec/nvec_power.c
+++ b/drivers/staging/nvec/nvec_power.c
@@ -338,7 +338,7 @@ static const struct power_supply_desc nvec_psy_desc = {
 };
 
 static int counter;
-static int const bat_iter[] = {
+static const int bat_iter[] = {
 	SLOT_STATUS, VOLTAGE, CURRENT, CAPACITY_REMAINING,
 #ifdef EC_FULL_DIAG
 	AVERAGE_CURRENT, TEMPERATURE, TIME_REMAINING,
-- 
2.11.0

